apparmor.d/profiles-m-r/pulseaudio

```diff
@@ -66,18 +69,72 @@ profile pulseaudio @{exec_path} {
   @{sys}/devices/virtual/dmi/id/{bios_vendor,board_vendor,sys_vendor} r,
   @{sys}/devices/system/node/ r,
   @{sys}/devices/system/node/node[0-9]/meminfo r,
-
-  owner @{run}/user/@{uid}/dconf/ w,
-  owner @{run}/user/@{uid}/dconf/user rw,
+  @{sys}/module/apparmor/parameters/enabled r,
```

Comment:
Is this really needed?

Reply:
It seems like pulseaudio is AppArmor-aware. We could block it, but I'm not sure about side-effects,I haven't dug the code. The sound is present.